test/unit/routesTest.js

```diff
@@ -308,28 +323,39 @@ describe('Roosevelt routes Section Test', function () {
 
     // on the message that tells us that the server has started, test the path that will run into a server error
     testApp.on('message', () => {
-      // indicator to tell whether or not we had a 503 come
-      let bool503 = false
-      for (let x = 0; x < 10; x++) {
-        request('http://localhost:43711')
-        .get('/HTMLTest')
-        .expect(200, (err, res) => {
-          if (err && res !== undefined && bool503 === false) {
-            assert.equal(res.status, 503)
-            bool503 = true
-            // sample the response text
-            const test1 = res.text.includes('503 custom test error page')
-            const test2 = res.text.includes('The server is either too busy or is under maintence, please try again later')
-            const test3 = res.text.includes('This is a test to see if we can make custom 503 controllers and pages')
-            // check to make sure that all specific pharses are there
-            assert.equal(test1, true)
-            assert.equal(test2, true)
-            assert.equal(test3, true)
-            testApp.kill()
-            done()
-          }
-        })
-      }
+      async.series([
+        function (cb) { request('http://localhost:43711').get('/HTMLTest').expect(200, (err, res) => { if (err) { cb(res, null) } else { cb(null, res) } }) },
```

Comment:
also i think there is probably a more intelligent way to structure this since each request is the same, can't we just fire off a loop that generates a bunch of these rather than copy/paste?